### ISSUE TITLE
`bbl plan` panics without a helpful error message when the user doesn't provide the `--iaas` flag

### ISSUE THREAD

**Issue by dsabeti** [cloudfoundry/bosh-bootloader#219]
Howdy! I was playing around with `bbl plan` today, and I forgot to add `--iaas gcp` to my command. I got the following:
```
$ bbl5 plan
step: generating terraform template
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x12f0c2c]

goroutine 1 [running]:
github.com/cloudfoundry/bosh-bootloader/terraform.Manager.Init(0x1ff1520, 0xc420161bc0, 0x0, 0x0, 0x0, 0x0, 0x1fe9ae0, 0xc42004be30, 0xc42016b3b0, 0x1fe71a0, ...)
	/go/src/github.com/cloudfoundry/bosh-bootloader/terraform/manager.go:95 +0x8c
github.com/cloudfoundry/bosh-bootloader/terraform.(*Manager).Init(0xc4200589c0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	<autogenerated>:1 +0xa5
github.com/cloudfoundry/bosh-bootloader/commands.Plan.Execute(0x1feec20, 0xc420058960, 0x1ff2160, 0xc420054a00, 0x1fed460, 0xc42016b490, 0x1fed4e0, 0xc4200f3960, 0x1fe97e0, 0xc4200f3980, ...)
	/go/src/github.com/cloudfoundry/bosh-bootloader/commands/plan.go:58 +0x2b9
github.com/cloudfoundry/bosh-bootloader/commands.(*Plan).Execute(0xc420058a20, 0xc42004be20, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	<autogenerated>:1 +0xc2
github.com/cloudfoundry/bosh-bootloader/application.App.execute(0xc420161c20, 0xc420013070, 0xd, 0x0, 0x7fff5fbffafd, 0x4, 0xc42004be20, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cloudfoundry/bosh-bootloader/application/app.go:86 +0x405
github.com/cloudfoundry/bosh-bootloader/application.App.Run(0xc420161c20, 0xc420013070, 0xd, 0x0, 0x7fff5fbffafd, 0x4, 0xc42004be20, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cloudfoundry/bosh-bootloader/application/app.go:34 +0x63
main.main()
	/go/src/github.com/cloudfoundry/bosh-bootloader/bbl/main.go:205 +0x44e9
```

It would be great if it could give a helpful error message. You could probably just use the same output for `bbl up` without the IaaS flag:
```
 $ bbl up


--iaas [gcp, aws, azure] must be provided or BBL_IAAS must be set
```

**Comment by rowanjacobs** [comment]
What version of bbl are you using?

**Comment by dsabeti** [comment]
@rowanjacobs 5.3.7. I realize it's a few weeks old: do you think newer versions have fixed this?

**Comment by rowanjacobs** [comment]
We were unable to recreate it on the latest dev version; I'll try with 5.4.0 and later and see which version fixed it.

**Comment by rowanjacobs** [comment]
Seems to be fixed in 5.4.0.
```
$ ~/Downloads/bbl-v5.4.0_osx plan
--iaas [gcp, aws, azure] must be provided or BBL_IAAS must be set
```

**Comment by dsabeti** [comment]
Thanks @rowanjacobs!
